eden/util/__init__.py

```diff
@@ -286,8 +287,15 @@ def describe(data_matrix):
 
 
 def iterator_size(iterable):
-    size = sum(1 for _ in iterable)
-    return size
+    """Length of an iterator."""
```

Comment:
iterator. it works with all iterables and not only with generators.

On Thu, Jul 7, 2016 at 10:25 AM, Björn Grüning <notifications@github.com>
wrote:

> In eden/util/__init__.py
> <https://github.com/fabriziocosta/EDeN/pull/111#discussion_r69868544>:
>
> > @@ -286,8 +287,15 @@ def describe(data_matrix):
> >
> >
> >  def iterator_size(iterable):
> > -    size = sum(1 for _ in iterable)
> > -    return size
> > +    """Length of an iterator."""
>
> iterator or generator?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fabriziocosta/EDeN/pull/111/files/961b7f957f1a2b41ec60a6792b57be6d3687dc21#r69868544>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQh4O-4FKWXsRsJAoy9q7GVlNwTu_dHTks5qTLfjgaJpZM4JG0qI>
> .
>



-- 
--------------------------------------
Gianluca Corrado
corrag@gmail.com